.cirrus.yml

```diff
@@ -681,11 +681,15 @@ podman_machine_task:
     # Required_pr_labels does not apply to non-PRs.
     # Do not run on tags, branches, [CI:BUILD], or [CI:DOCS].
     only_if: *not_tag_branch_build_docs
+    only_if: *is_pr
+    # FIXME: label driven tests are unpredictable in terms of triggering
+    # for machine tasks. Run them on all PRs for now.
+    #
```

Comment:
Please remove 'FIXME'

Reply:
Why remove the FIXME? That looks like literally the perfect example of a FIXME: something that should work but is broken, and nobody knows why, and nobody has time to figure it out today.

Reply:
Because I think we should just give up and run this for every PR.  Nearly any change to podman code could impact it's functionality, so best not treat it with kid-gloves any more.  At least that was my reasoning.

Reply:
Also...were someone to try re-enabling it, they would also need to remember to add it back to the exclude list of `cirrus_yaml_test.py` and also yank it as a dependency from success.  It's just too much trouble, not worth it.

Reply:
done